### PR TITLE
Configure a default TOC for a given Version

### PR DESCRIPTION
**Pull request by jacobwegner**
- Retrieve a default TOC if `defaultTocUrn ` has been defined for a Version (ATLAS PR: https://github.com/scaife-viewer/sv-mini-atlas/pull/6)
- Show's the root TOC by default 
- Configure the TOC endpoint in the store (so it can be overridden in a particular project, or in this case, a particular branch of a project as deployed to Netlify)

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ import { WIDGETS_NS } from '@/store/constants';
 const state = {
   readerTextSize: 'md',
   readerTextWidth: 'normal',
+  // TODO: Determine if there is a better way to override this
```
Comment (jacobwegner) [inline src/store/index.js]: see discussion on https://github.com/scaife-viewer/sv-mini/pull/10/files#r383517451

**Comment by jacobwegner** [comment]
[As a curator, I want to define the default TOC for a given URN's version](https://trello.com/c/xVl0uXmG/56-as-a-curator-i-want-to-define-the-default-toc-for-a-given-urns-version)

File: src/widgets/TOCWidget.vue [hunk 2]
```diff
@@ -18,28 +19,39 @@
...
+        return this.$store.getters[`${WIDGETS_NS}/metadata`];
+      },
+      defaultTocUrn() {
+        return this.metadata && this.metadata.defaultTocUrn
+          ? this.metadata.defaultTocUrn
+          : this.rootTocUrn;
+      },
       endpoint() {
-        return 'https://mini-stack-a-feature-se-j47yu0.herokuapp.com';
+        return this.$store.state[`${WIDGETS_NS}`].tocEndpoint;
```
Comment (jhrr) [inline src/widgets/TOCWidget.vue]: Probably want to define a getter (like `this.$store.getters[`${WIDGETS_NS}/metadata`]` above) for this, just to be consistent with the other widgets

Comment (jhrr) [reply src/widgets/TOCWidget.vue]: Ah ok nevermind, looks like you're handling this value in a different way than a normal state value so this is probably moot.

**Comment by jacobwegner** [comment]
can be previewed at:

http://feature--toc-lookahead--sv-mini.netlify.com/

(but I'll leave it up to @jhrr to actually get a proper `sv-mini` PR)

**Pull request merged**